src/System.Runtime/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,4 @@
+MembersMustExist : Member 'System.Type.IsSecurityCritical.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
@joperezr Yes, my change was merged with this new commit. Thanks for pointing it out. Although, my change only included System.Reflection.Pointer and the three System.Type APIs are not yet available, since there is no VM implementation for them.
Will remove the baseline for System.Reflection.Pointer.